vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -103,6 +103,7 @@
     </Link>
     <Midl>
       <AdditionalIncludeDirectories>$(ReactNativeWindowsDir)\Microsoft.ReactNative;$(ReactNativeWindowsDir)\Microsoft.ReactNative.Cxx;</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>CORE_ABI</PreprocessorDefinitions>
```

Comment:
Should we include `; %(PreprocessorDefinitions)` here?
Now if one of the imports add some preprocessor definitions, they will be removed, instead of appendnig.

Reply:
Yes, fixed in second iteration of the change.